recipes/renode-cli/build.sh

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env bash
+
+set -o xtrace -o nounset -o pipefail -o errexit
+
+update_antmicro_submodule() {
+  local commit=$1
+  local module=$2
+
+  git clone https://github.com/antmicro/${module}.git
+  pushd ${module}
+    git fetch origin ${commit}
+    git submodule update --init --recursive
+  popd
+}
+
+mkdir -p ${PREFIX}/bin
+mkdir -p ${PREFIX}/libexec/${PKG_NAME}
+export PATH="${DOTNET_ROOT}/dotnet:${PATH}"
+
+install_prefix="${PREFIX}/opt/${PKG_NAME}"
+
+dotnet_version=$(dotnet --version)
+framework_version=${dotnet_version%.*}
+
+mkdir -p ${SRC_DIR}/src/Infrastructure/
+mv renode-infrastructure/* ${SRC_DIR}/src/Infrastructure/ && rm -rf renode-infrastructure
+
+mkdir -p ${SRC_DIR}/lib
+pushd ${SRC_DIR}/lib
+  git clone https://github.com/renode/renode-resources.git
+  pushd renode-resources && git submodule update --init --recursive && popd
+  mv renode-resources resources
+
+  update_antmicro_submodule b1d3d03d602581fc2bed6db586b5e5c3388456c7 AntShell
+  update_antmicro_submodule e7bfa5873f2300e6e87c185f466e227762dbf4b2 BigGustave
+  update_antmicro_submodule b0c2a820f28a7bdedb85575bfca6447c9e7fa955 CxxDemangler
+  update_antmicro_submodule de4e4f6ffab555771285cb810f17f61cfd38ef39 ELFsharp
+  rm -rf ELFSharp && mv ELFsharp ELFSharp
+  update_antmicro_submodule e379e8ae696676afffed2f33dd8083855af00f2f FdtSharp
+  update_antmicro_submodule 33e5ab24eaaab488e9a94f1a40d5d6ae2f7f02f1 InpliTftpServer
+  update_antmicro_submodule c6514e99f2c35afec083b9f4a7eec3408c0081d1 Migrant
+  update_antmicro_submodule 2fe74fd257f6d6f86076c100952178f347098b3d Packet.Net
+  update_antmicro_submodule 33d24f1307d267c34b7f1439bc159a8825c5aa3d bc-csharp
+  update_antmicro_submodule bde21d04fbfc540989b7a0ac13a54eae8b756994 cctask
+  update_antmicro_submodule e8c2051dec56c5ccd3a4927b07a5740f34eed8c8 options-parser
+  update_antmicro_submodule 71af57ef4fec29e416f48160b8918057a58548a9 termsharp
+popd
```

Comment:
I believe you'll be able to remove all of these when you've switch to the new source archive I suggested.

Reply:
Would you accept to be a co-maintainer. I can tell that there are a lot of updates in `master`, the next release might be a tough one as well

Reply:
You can add me as co-maintainer, this will be a useful recipe for figuring mixed C#/C packages in the future.